types/pdfjs-dist/index.d.ts

```diff
@@ -260,58 +261,128 @@ interface PDFDocumentProxy {
      **/
     fingerprint: string;
 
-    /**
-     * True if embedded document fonts are in use.  Will be set during rendering of the pages.
-     **/
-    embeddedFontsUsed(): boolean;
-
     /**
      * @param number The page number to get.  The first page is 1.
      * @return A promise that is resolved with a PDFPageProxy.
      **/
-    getPage(number: number): PDFPromise<PDFPageProxy>;
+    getPage(number: number): Promise<PDFPageProxy>;
+
+    /**
+     * @param ref The page reference. Must have the `num` and `gen` properties.
+     * @return A promise that is resolved with the page index that is associated with the reference.
+     */
+    getPageIndex(ref: {num: number, gen: number}): Promise<number>;
 
     /**
      * TODO: return type of Promise<???>
      *  A promise that is resolved with a lookup table for mapping named destinations to reference numbers.
      **/
-    getDestinations(): PDFPromise<any[]>;
+    getDestinations(): Promise<any[]>;
+
+    /**
+     * TODO: return type of Promise<???>
```

Comment:
Are these TODOs still valid?

Reply:
Yeah. Unfortunately the documentation in PDF.js doesn't specify clearly what these methods return other than a promise, so I was unable to determine what the real type should be

Reply:
I guess the TODO should at least be taken out of the JSDoc comment so users don’t see it